fcrepo-http-api/src/main/resources/views/common.js

```diff
@@ -94,7 +94,15 @@
       const url = this.href;
       http('HEAD', url, function(res) {
         if (res.status >= 400 || res.getResponseHeader('Link') == null) {
-          location.href = url;
+          var newLocation = url;
```

Comment:
There are a number of issues with this code, and I don't actually see a good resolution.

First, to @awoods point -- the `XMLHttpRequest` will transparently follow 3xx redirects, so there is no way to test for, say, a 307 response. 

Secondly, with the redirect in place, if the remote resource doesn't have CORS headers set up correctly for the request, you'd get a preflight error anyway.

Third, the comment:

> ... to the external resource: therefore there is no Link header.

How can you guarantee that there is no `Link` header on a resource that is outside of Fedora's domain? That is very shaky ground to stand on.

Fourth, testing for the presence of `/fcr:metadata` is not a particularly "LDP" way of doing things. Yes, it works with this particular implementation of Fedora, but that's all one could claim.

That said, I don't want to stand in the way of addressing this ticket, so feel free to merge this code as-is.